src/stretchy.js

```diff
@@ -227,13 +234,12 @@ const svgSpan = function(group, options) {
             } else {
                 span = buildCommon.makeSpan([widthClass], [svgNode], options);
                 span.style.height = height + "em";
-                spans.push(span);
             }
+            spans.push(span);
```

Comment:
Why did this get moved out of the `else` block?

Reply:
So that I can access it in the `span =` assignment below. Flow wasn't convinced that `span` was guaranteed to be assigned otherwise.

Reply:
That seems really weird as the `span` assignment is in both the `if` and `else` block.  I'll check out the branch and play around with it.

Reply:
I agree, I don't like that either. I can actually fix that really easily by using a block local `span` variable to make that clearer.